### PR TITLE
Update main.tf

### PR DESCRIPTION
**Pull request by haytham0123**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

### Motivation

- Resolves #<issue-number>

### More

- [ ] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/eks-blueprints-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/docs) for this feature
- [ ] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs require a new example and/or doc page. In general:
- Use an existing example when possible to demonstrate a new addons usage
- A new docs page under `docs/add-ons/*` is required for new a new addon

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: examples/node-groups/managed-node-groups/main.tf [hunk 1]
```diff
@@ -126,7 +126,7 @@ module "eks_blueprints" {
       public_ip         = false # Use this to enable public IP for EC2 instances; only for public subnets used in launch templates
 
       http_endpoint               = "enabled"
-      http_tokens                 = "optional"
+      http_tokens                 = "required"
```
Comment (FernandoMiguel) [inline examples/node-groups/managed-node-groups/main.tf:129]: why this change?

**Comment by bryantbiggs** [comment]
thank you but we won't be making this change at this time. Next time, please start with an issue and we can discuss the problem we are trying to solve for and potential solutions before a PR

File: examples/node-groups/managed-node-groups/main.tf (hunk 1 above)
Comment (haytham0123) [reply examples/node-groups/managed-node-groups/main.tf:129]: to introduce IMDSv2 which is a security best practice

Comment (haytham0123) [reply examples/node-groups/managed-node-groups/main.tf:129]: Or we can have it as a variable please instead of being hard coded

Comment (bryantbiggs) [reply examples/node-groups/managed-node-groups/main.tf:129]: in the next [iteration of blueprints](https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1319), we will be utilizing the `terraform-aws-eks` module for cluster and node group creation and this module already has the default values set that you are looking for https://github.com/terraform-aws-modules/terraform-aws-eks/blob/a8e0de0b3f3a3e27232b35e9ed8814549273bced/node_groups.tf#L2-L6
